### DISCUSSION TITLE
v-dialog

### DISCUSSION THREAD

**Discussion by Chimi4** [vuetifyjs/vuetify#12236] [General]
How to minimize and maximize v-dialog  and we should able to edit other components after minimizing the dialog box?

**Comment by mtermoul** [comment]
v-dialog have a `width` prop which allow you to control the width. For instance you can have 
```
<v-dialog v-model="showDialog" width="400px" ...>
or 
<v-dialog v-model="showDialog" width="800px" ...>
```
You can also use the `full-screen` property to make full screen as the the following:
```
<v-dialog v-model="showDialog" full-screen ...>
```

Now if you want to open/close the dialog after you opened it initially which will give you the effects of maximize/minimize then you just set the `showDialog` variable to true or false like the code below.

```
<script>
...
methods: {
   onButtonClick() {
      this.showDialog = !this.showDialog;
    }
}
...
</script>
```

By the way you can have answers quickly if you post your question in the Discord support channel for Vuetify https://discord.com/invite/s93b7Fv

Happy coding.
